x/wasm/keeper/handler_plugin_encoders_test.go

```diff
@@ -421,13 +419,50 @@ func TestEncoding(t *testing.T) {
 		"stargate encoded invalid typeUrl": {
 			sender: addr2,
 			srcMsg: wasmvmtypes.CosmosMsg{
-				Stargate: &wasmvmtypes.StargateMsg{
+				Any: &wasmvmtypes.AnyMsg{
 					TypeURL: "/cosmos.bank.v2.MsgSend",
 					Value:   bankMsgBin,
 				},
 			},
 			expError: true,
 		},
+	}
+	encodingConfig := MakeEncodingConfig(t)
+	for name, tc := range cases {
+		t.Run(name, func(t *testing.T) {
+			var ctx sdk.Context
+			encoder := DefaultEncoders(encodingConfig.Codec, wasmtesting.MockIBCTransferKeeper{})
+			res, err := encoder.Encode(ctx, tc.sender, "", tc.srcMsg)
```

Comment:
probably the variable could be removed
```suggestion
			res, err := encoder.Encode(sdk.Context{}, tc.sender, "", tc.srcMsg)
```